pkg/kubeapiserver/admission/config.go

```diff
@@ -141,13 +146,13 @@ func GetAdmissionPluginConfigurationFor(pluginCfg componentconfig.AdmissionPlugi
 
 // GetAdmissionPluginConfiguration takes the admission configuration and returns a reader
 // for the specified plugin.  If no specific configuration is present, we return a nil reader.
-func GetAdmissionPluginConfiguration(cfg *componentconfig.AdmissionConfiguration, pluginName string) (io.Reader, error) {
```

Comment:
> A pity we cannot easily move AdmissionConfiguration as apis/componentconfig into genericapiserver and extends that group from kubeapiserver.

I agree, but instead creating a targeted group he re-used an existing one, so it can't move together.  It's something we should take care of, but it may produce a forever incompatibility with kube itself. :(